audio_output_switch

```diff
@@ -1,3 +1,22 @@
 #!/bin/sh
 
-pactl set-default-sink "$(pactl list short sinks | awk '{print $2}' | dmenu -i -l 5 -c -p "Output:")" && notify-send "Audio switched!" || exit 0
+# Create a list of sinks with pretty names
+sinks=$(pactl list short sinks)
+options=$(echo "$sinks" | while read -r line; do
+    name=$(echo "$line" | awk '{print $2}')
+    description=$(pactl list sinks | grep -A 20 "$name" | grep "Description:" | head -n1 | cut -d ':' -f2- | sed 's/^ //')
+    echo "$description|||$name"
+done)
```

Comment:
Instead of capturing the sinks and looping through them later, you can do it all at once.

```shell
options=$(while read -r name; do
    description=$(pactl list sinks | awk -v name="$name -F ':' '$0 ~ /Name:/ && $0 ~ name {f=1}; f == 1 && $0 ~ /Description:/ { print $2; exit }')
    echo "$description|||$name"
done < <(pactl list short sinks | awk '{ print $2 }'))
```

This works in `bash`. I would assume it works in `sh`. 

Also instead of the splitting on the pipe character you could even use an associative array. I don't believe that `sh` is that fancy though.